vscode/src/lspClient.ts

```diff
@@ -204,6 +198,14 @@ export function activateLsp(context: ExtensionContext, lspCommand: Array<string>
     }
   }
 
+  async function didOpenTextDocument(document: TextDocument): Promise<void> {
+    try {
+      await didOpenTextDocumentThrow(document);
+    } catch(e) {
+      Window.showErrorMessage("" + e);
```

Comment:
Other question: what would happen if we informed vscode about the error?
Would it be ignored? ...

Reply:
The error is ignored, I don't think vscode handles errors on events.